src/Language/blockStringValue.php

```diff
@@ -12,16 +12,16 @@
  */
 function blockStringValue(string $rawString): string
 {
-    $lines     = preg_split("/\r\n|[\n\r]/", $rawString);
-    $lineCount = count($lines);
-
+    $lines        = \preg_split("/\r\n|[\n\r]/", $rawString);
+    $lines        = false === $lines ? [] : $lines;
+    $lineCount    = \count($lines);
     $commonIndent = null;
 
     for ($i = 1; $i < $lineCount; $i++) {
         $line   = $lines[$i];
         $indent = leadingWhitespace($line);
 
-        if ($indent < mb_strlen($line) && ($commonIndent === null || $indent < $commonIndent)) {
+        if ($indent < \mb_strlen($line) && ($commonIndent === null || $indent < $commonIndent)) {
```

Comment:
Not a big deal but should we change to `null === $commonIndent` for consistency?

Reply:
Fixed this as well.